lesson_7/mihaela_vaida/print_register_dec.py

```diff
@@ -0,0 +1,21 @@
+
+print_registry = []
+def register(fnc):
+    
+    
+    def print_register(*args):        
+        print_registry.append(fnc.__name__)
+        print(print_registry)
+    return print_register   
+        
+
+
+@register
+def greet(name):
+    return "Greetings {}!".format(name)
+
+def say_hello(name):
+    return "Hello {}!".format(name)
+@register
+def say_goodbye(name):
+    return "Goodbye {}!".format(name)
```

Comment:
Please add at the end a use case for the decorator and the decorated functions: print the registry and call the decorated functions.

Reply:
Done